src/print-prescription/prescription-printout.component.tsx

```diff
@@ -0,0 +1,121 @@
+import { Layer, StructuredListBody, StructuredListCell, StructuredListRow, StructuredListWrapper } from '@carbon/react';
+import { formatDate, parseDate, useSession } from '@openmrs/esm-framework';
+import React from 'react';
+import { useTranslation } from 'react-i18next';
+import { type DosageInstruction, type MedicationRequestBundle, type Quantity } from '../types';
+import {
+  getDosageInstruction,
+  getMedicationDisplay,
+  getMedicationReferenceOrCodeableConcept,
+  getQuantity,
+  getRefillsAllowed,
+} from '../utils';
+import styles from './print-prescription.scss';
+
+type PrescriptionsPrintoutProps = {
+  medicationrequests: Array<MedicationRequestBundle>;
+  excludedPrescription: Array<string>;
+};
+
+const PrescriptionsPrintout: React.FC<PrescriptionsPrintoutProps> = ({ medicationrequests, excludedPrescription }) => {
+  const { t } = useTranslation();
+  const {
+    sessionLocation: { display: facilityName },
+  } = useSession();
+  const patient = medicationrequests[0]?.request?.subject;
+  return (
+    <Layer className={styles.printOutContainer}>
+      <StructuredListWrapper>
+        <StructuredListBody>
+          <StructuredListRow head>
+            <StructuredListCell head>
+              <br />
+              <br />
+              <p className={styles.printoutTitle}>Prescription Instructions</p>
+              {patient && (
+                <p className={styles.faintText} style={{ textAlign: 'center' }}>
+                  {patient.display
+                    .match(/^([A-Za-z\s]+)\s\(/)
+                    ?.at(1)
+                    ?.toUpperCase()}
```

Comment:
I don't think we should manipulate the patient display here... I hope we have some sort of existing utility formatting method we can use... @denniskigen @ibacher ?

Reply:
extracted it into a utility function